tests/e2e/data/config-taquito-test-environment-invalid-config-network-url.json

```diff
@@ -17,61 +20,39 @@
 				"local"
 			]
 		},
-		"test": {
-			"networks": ["ithacanet"],
+		"testing": {
+			"networks": [
+				"ghostnet"
+			],
 			"sandboxes": []
 		}
 	},
 	"sandbox": {
 		"local": {
 			"label": "Local Tezos Sandbox",
-			"protocol": "Psithaca2MLRFYargivpo7YvUr7wUDqyxrdhC5CQq78mRvimz6A",
-			"rpcUrl": "http://localhost:20050"
+			"protocol": "PtKathmankSpLLDALzWw7CGD2j2MtyveTwboEYokqUCP4a1LxMg",
+			"rpcUrl": "http://localhost:20030"
 		}
 	},
 	"network": {
-		"ithacanet": {
-			"label": "ithacanet",
-			"rpcUrl": "https://invalid.test",
-			"protocol": "Psithaca2MLRFYargivpo7YvUr7wUDqyxrdhC5CQq78mRvimz6A",
+		"ghostnet": {
+			"label": "ghostnet",
+			"rpcUrl": "https://ghostnetbet.ecadinfrastructure.com",
```

Comment:
What is this URL?

Reply:
It's meant to be a bogus URL that won't work to make sure Taqueria will tell you that something is wrong